test/parsing/ferguson/trailing-comma.chpl

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
It would be good to extend the test to include the other two cases, domains and associative arrays.